BelatrixConnect_AutomationDSL/src/test/java/com/belatrix/connect/framework/DSL.java

```diff
@@ -13,9 +15,45 @@ public DSL(WebDriver driver) {
 
 	/**
 	 * @param locator
-	 */
+	 **/
```

Comment:
Se especifico la funcion del metodo:

	/**
	 * This method use for send click to Element
	 * @param By locator
	 **/